provisioning/baremetal_pod.go

```diff
@@ -449,6 +496,7 @@ func createContainerMetal3BaremetalOperator(images *Images, config *metal3iov1al
 			},
 			buildEnvVar(deployKernelUrl, config),
 			buildEnvVar(deployRamdiskUrl, config),
+			buildEnvVar(deployIsoUrl, config),
```

Comment:
Given what we discussed last week, I wonder if we should skip this part, so that Ironic keeps building an ISO itself, using the kernel and ramdisk provided. Maybe as a follow-up patch if this one is already tested to work with virtual media.